src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -26,13 +26,12 @@ public class SdkDirectoryWorkloadManifestProvider : IWorkloadManifestProvider
 
         private readonly WorkloadSet? _workloadSet;
 
-        public SdkDirectoryWorkloadManifestProvider(string sdkRootPath, string sdkVersion, string? userProfileDir, string? requestedWorkloadSet = null)
-            : this(sdkRootPath, sdkVersion, Environment.GetEnvironmentVariable, userProfileDir, requestedWorkloadSet)
+        public SdkDirectoryWorkloadManifestProvider(string sdkRootPath, string sdkVersion, string? userProfileDir, string? globalJsonPath)
```

Comment:
A lot of places seem to call this thing without a globalJsonPath... is it a good idea to make it an optional parameter with a null default instead?

Reply:
I think the places that pass null for this are all in the tests.  Since I was adding a new parameter I wanted to make sure to update all the code that calls it to account for the new parameter.